Representor/HTTP/HTTPTransition.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  HTTPTransition.swift
+//  Representor
+//
+//  Created by Kyle Fuller on 23/01/2015.
+//  Copyright (c) 2015 Apiary. All rights reserved.
+//
+
+import Foundation
+
+/** An implementation of the Transition protocol for HTTP. */
+public struct HTTPTransition : Transition {
+    public typealias Builder = HTTPTransitionBuilder
+
+    public let uri:String
+
+    /// The HTTP Method that should be used to make the request
+    public let method:String = "POST"
+    /// The suggested contentType that should be used to make the request
+    public let contentType:String?
```

Comment:
After thinking about this some more, maybe we should extend on Siren's understanding of this and it should become an array so we can have suggest multiple types in order of preference.

```swift
/// An array of suggested content types to use when making a request. Ordered by preference.
public let suggestedContentTypes:[String]
```

Opinions?